documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -2074,6 +2074,150 @@ Constant::
 If the mapping definition does not include the `header:` or `value:` prefixes, when {prodname} detects instances of the specified value in incoming messages, it treats them as constants, and uses them as-is.
 Use this option when you want to map the value of a specific static constant to an attribute in the `qna.yml` file.
 
+
+==== Apache Iceberg
+
+The {prodname} Apache Iceberg connector is a sink that runs in {prodname} Server. It consumes database change events from a source {prodname} connector and writes them to Apache Iceberg tables.
+This allows for direct replication of a Database (Change Data Capture events) to Iceberg tables on cloud storage or HDFS, eliminating the need for intermediate systems like ETL Tool, Spark or other streaming platforms.
+
+The {prodname} Iceberg connector supports both `upsert` and `append` modes for data replication. When event and key schema information is enabled, the destination Iceberg tables are automatically created upon the initial startup.
+
+The connector provides the following features:
+
+* Upsert and Append modes.
+* Optimizing batch size.
+* Customizable table naming.
+* Offset and Schema History storage in Iceberg.
+* Automatic schema evolution.
+
+[NOTE]
+====
+- It is recommended to use `ExtractNewRecordState` transformation (event flattening). Key features like `upsert` mode and automatic primary key detection depend on the flattened event structure produced by this SMT. Without it, the connector can only operate in a more limited append-only mode.
+
+- The Debezium Server sink for Apache Iceberg is a community-maintained, open-source project. Please note that it is maintained in a separate repository, which you can find here: link:https://github.com/memiiso/debezium-server-iceberg[Debezium Server Iceberg GitHub]
```

Comment:
@jpechane its ready for review. Is there anything else to add remove? 

added here a note and link to the github repo, please feel free to suggest how to highlight it.